src/main/java/frc/robot/commands/LiftToHeight.java

```diff
@@ -25,6 +23,25 @@ public LiftToHeight(HangSubsystem hangSubsystem) {
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
-    hangSubsystem.liftToHeight();
+    hangSubsystem.resetEncoder();
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    hangSubsystem.liftLift();
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+    hangSubsystem.stopLifting();
```

Comment:
Maybe we rename this (minus `ing` since we're adding the verb to the front)

```suggestion
    hangSubsystem.stopLift();
```